packages/core/src/test/awsService/ec2/activation.test.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as vscode from 'vscode'
+import * as sinon from 'sinon'
+import { assertTelemetry } from '../../testUtil'
+import { Ec2InstanceNode } from '../../../awsService/ec2/explorer/ec2InstanceNode'
+import { Ec2ParentNode } from '../../../awsService/ec2/explorer/ec2ParentNode'
+import { Ec2Client } from '../../../shared/clients/ec2Client'
+import { Ec2ConnectionManager } from '../../../awsService/ec2/model'
+import { PollingSet } from '../../../shared/utilities/pollingSet'
+
+describe('ec2 telemetry', function () {
```

Comment:
Telemetry is generally just logging that happens when code runs. Asserting telemetry is thus asserting that code was executed. So in most cases, we rarely need explicit "telemetry tests" (except for the telemetry module itself). 

Maybe there's not much to test right now, but these tests can be positioned as general tests for ec2 or whatever, not "telemetry tests". That makes it clear for future readers that this is where new tests for this module should go.

```suggestion
describe('ec2 activation', function () {
```